TicketsDemo/App_Data/Carriage.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" standalone="yes"?>
```

Comment:
xml files are resources
log files are resources

both produced/consumed by applications during it's lifetime.
Both are NOT a part of the codebase.

Use .gitignore file in order to exclude these files from the git repository.